packages/core/contracts/oracle/implementation/VotingV2.sol

```diff
@@ -702,7 +731,6 @@ contract VotingV2 is
             "Cannot commit inactive request"
         );
 
-        priceRequest.lastVotingRound = currentRoundId;
```

Comment:
the change here is VERY subtle: previously if a request had been rolled we'd update the last voting round here. now, it happens within the `_updateCumulativeSlashingTrackers` function.